src/components/messenger/autocomplete-members/index.tsx

```diff
@@ -29,25 +30,32 @@ export class AutocompleteMembers extends React.Component<Properties, State> {
     }
 
     const items = await this.props.search(searchString);
-
-    this.setState({ results: items.map(itemToOption) });
+    const itemsToOption = items.map(itemToOption);
```

Comment:
I think these would be `options` now that we've mapped "items" to "options".

Reply:
makes sense